google-cloud-monitoring/.rubocop.yml

```diff
@@ -3,6 +3,7 @@ AllCops:
     - "integration/**/*"
     - "google-cloud-monitoring.gemspec"
     - "lib/google/monitoring/**/*"
+    - "lib/google/cloud/monitoring/**/*"
```

Comment:
Should we be concerned that this pattern will potentially exclude future hand-written code from rubocop checks?